kiteco/KiteSublime
# Sublime Onboarding with Command Palette

Addresses Sublime for https://github.com/kiteco/kiteco/issues/10456 for the updated spec.
Should merge for release at JS public launch.

### What was done
Reverted beta specific changes from https://github.com/kiteco/KiteSublime/pull/72
Added commands to open Go and Javascript tutorials
Simplified help notification logic.

### Known Issues
If Kite app isn't running, the Python onboarding file may not open properly since the call open the onboarding file doesn't wait for the Kite app to open. It defaults to the screenshotted dialog below.

### QA
Check that:

1. Opening Sublime with `show_help_dialog: true` or missing in `KiteSublime.sublime-settings -- User` attempts to open the Python onboarding file.
2. Commands for `Kite: Go Tutorial` and `Kite: Javascript Tutorial` are present in the command palette. (Note that these will fail to pull an onboarding file from kited at the moment as those files do not exist).
3. The `Kite: <lang> Tutorial` options show this dialog when the Kite app is not running:
<img width="588" alt="Screen Shot 2020-04-24 at 11 44 50 AM" src="https://user-images.githubusercontent.com/18232816/80252957-38093800-862d-11ea-9cb1-fe83d8474957.png">
